src/pages/About.vue

```diff
@@ -1,15 +1,138 @@
+              <p class="is-size-5">
+                OCF Staff meet to discuss technology, learn from each other, and
+                work on OCF projects. Drop-in visitors are always welcome,
+                especially those interested in joining the staff team!
+              </p>
+              <p class="is-size-4">
+                <strong>When:</strong> Every Monday at 8:10pm
+              </p>
+              <p class="is-size-4">
+                <strong>Where: </strong>
+                <a href="https://ocf.io/meet">Google Meet</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="is-divider"></div>
+    <section class="section">
+      <div class="container">
+        <div class="columns is-vcentered reverse-row-order">
+          <div class="column is-half">
+            <transition name="fade" mode="out-in">
+              <img
+                :src="servicesImages[servicesActiveImage]"
+                :key="servicesImages[servicesActiveImage]"
+                data-toggle="tooltip"
+                width="800"
+                height="600"
+                data-placement="bottom"
+                title="goTop"
+                alt=""
+              />
+            </transition>
+          </div>
+          <div class="column is-half">
+            <div class="columns is-centered is-vcentered">
+              <div class="column is-three-quarters">
+                <div class="content">
+                  <h2 class="title is-2 has-text-centered">
+                    Services We Provide
+                  </h2>
+
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="servicesActiveImage = 0"
+                    :class="{ 'active-card': servicesActiveImage == 0 }"
+                  >
+                    Web &amp; Email Hosting
+                  </p>
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="servicesActiveImage = 1"
+                    :class="{ 'active-card': servicesActiveImage == 1 }"
+                  >
+                    UNIX Shell Accounts
+                  </p>
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="servicesActiveImage = 2"
+                    :class="{ 'active-card': servicesActiveImage == 2 }"
+                  >
+                    Free Printing for Members
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="is-divider"></div>
+    <section class="section">
+      <div class="container">
+        <div class="columns is-vcentered">
+          <div class="column is-half">
+            <img
+              src="http://3.bp.blogspot.com/-l5WiG6Nci34/UVKfPBediVI/AAAAAAAAABk/48mdp61vr5k/s1600/Emperor-Penguins-Couple.jpg"
```

Comment:
Not sure why there is an `http` instead of `https` here. Maybe also downloading this image and storing it instead of loading it from another site (takes bandwidth on their end).

Reply:
These are mostly placeholder images; I'm hoping we can make some graphics that represent the OCF

Reply:
Perhaps use a `placeholder.jpg` to signify this when others are working on it

Reply:
Done!